desktop/apps/consignments2/components/describe_work_desktop/index.js

```diff
@@ -109,18 +113,21 @@ function DescribeWorkDesktop (props) {
             <Field name='height' component={renderTextInput}
               item={'height'}
               label={'Height*'}
+              warn={numberWarning}
```

Comment:
Heh -- it would have been a fun exercise, but can you imagine having to write all this from scratch?